homeassistant/components/cast/media_player.py

```diff
@@ -423,6 +575,14 @@ def async_cast_removed(discover: ChromecastInfo):
             async_cast_removed)
         self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_stop)
         self.hass.async_create_task(self.async_set_cast_info(self._cast_info))
+        for info in self.hass.data[KNOWN_CHROMECAST_INFO_KEY]:
+            if _is_our_dynamic_group(self._cast_info, info):
+                _LOGGER.debug("[%s %s (%s:%s)] Found dynamic group: %s",
+                              self.entity_id, self._cast_info.friendly_name,
+                              self._cast_info.host, self._cast_info.port, info)
+                self.hass.async_create_task(
+                    self.async_set_dynamic_group(info))
```

Comment:
Do we need to do any wrapping so we catch exceptions properly ?

Reply:
What kind of wrapping?